internal/store/elasticsearch/discovery_repository_test.go

```diff
@@ -31,7 +31,7 @@ func TestDiscoveryRepositoryUpsert(t *testing.T) {
 		)
 		require.NoError(t, err)
 
-		repo := store.NewDiscoveryRepository(esClient, log.NewNoop())
+		repo := store.NewDiscoveryRepository(esClient, log.NewNoop(), time.Duration(10)*time.Second)
```

Comment:
let's make it simpler
```suggestion
		repo := store.NewDiscoveryRepository(esClient, log.NewNoop(), time.Second*10)
```